### DISCUSSION TITLE
Continuing/Restarting a BespokeFit stopped by a job scheduler

### DISCUSSION THREAD

**Discussion by 123cpb** [openforcefield/openff-bespokefit#346] [Q&A]
Hello 

Is there a way to continue a Bespokefit run that is stopped due to the run going over the allowed time on a job script.  

The QC data generation stage of the molecular fragments I am trying to parameterize (at B3LYP-D3BJ/DZVP) takes longer than I am able to request on a HPC.  

Any help with this would be much appreciated

**Comment by j-wags** [comment]
Unfortunately, I don't think this is easily possible. The tooling isn't made for checkpointing and restarts. 

If you're an advanced user(and with your HPC admin's blessing) you might consider running the BespokeFit executor on the HPC head node (or somewhere else where you can run long processes and connect to compute nodes), and distributed workers on the time-limited compute nodes. Instructions for that are [here](https://docs.openforcefield.org/projects/bespokefit/en/latest/users/bespoke-executor.html#distributed-workers) but the details of networking are often complex enough that you'll need to be a somewhat expert user and it will likely require your sysadmin's assistance.
